src/media/css/app-list.styl

```diff
@@ -117,9 +117,19 @@ section.langpacks > div {
         &:nth-child(2n) {
             border-right: 0;
             padding-right: 0;
+
+            +rtl() {
+                border-right: $border;
+                padding: 20px 50px 20px 0;
```

Comment:
Actually `app-list-app { padding: 20px 50px; };` is already being set. Could move the `padding-right` rule into an `+ltr()` block.

```styl
&:nth-child(2n) {
  +ltr() {
    border-right: 0;
    padding-right: 0;
  }
  +rtl() {
    border-right: $border;
    padding-left: 0;
  }
}